.idea/prettier.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PrettierConfiguration">
+    <option name="myConfigurationMode" value="AUTOMATIC" />
+    <option name="myFilesPattern" value="**/*.{js,ts,jsx,tsx,cjs,cts,mjs,mts,vue,astro,html,xhtml}" />
```

Comment:
> However, with the default settings, it won't run on .html files.

@weegeekps Can this be `**/*` or just `*`?
If not you should add `css` and `md` to the extension list.
I would've expected the extension to rely on the `.prettierignore` file to dictate what it should and shouldn't run against. If it doesn't do this or if it only does the ignore check _after_ this list of files then you need those extensions as well to match every file we normally lint

Reply:
IIRC, it's not advisable to use `**/*` for performance reasons. I added `css` and `md` to the list of extensions in the glob pattern.